### PR TITLE
Fix typo in README.md

### PR DESCRIPTION
**Pull request by izeye**
This PR polishes the `README.md` a bit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,15 +60,14 @@ We use http://www.apache.org/licenses/LICENSE-2.0
 ### Spring Integration
 
 As the spring projects
-[removed the embed mongo support in 2.7.0](https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.7-Release-Notes#springmongodbembeddedfeatures-configuration-property-removed)
```
Comment (michaelmosmann) [inline README.md:63]: Yes, they updated the comment in their new release .. but they started removing the dependency with version 2.7.x. So i would not change that.

File: README.md [hunk 2]
```diff
@@ -60,15 +60,14 @@ We use http://www.apache.org/licenses/LICENSE-2.0
...
+[removed the embed mongo support in 3.0.0](https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.0-Migration-Guide#embedded-mongodb)
 you should consider to use one of these integration projects.
 It should behave mostly like the original spring integration, but there are some minor differences:
 - version in 'spring.mongodb.embedded.version' is used in package resolver and is not matched against version enum.
 - 'spring.mongodb.embedded.features' is not supported (not the way to change the config of mongodb)
 
 If you have any trouble in using them fell free to create an issue.
 
-- [Spring 2.5.x](https://github.com/flapdoodle-oss/de.flapdoodle.embed.mongo.spring/tree/spring-2.5.x)
```
Comment (michaelmosmann) [inline README.md:71]: Spring 2.5.x and 2.6.x support is still there. So i would not change that.

File: README.md [hunk 3]
```diff
@@ -60,15 +60,14 @@ We use http://www.apache.org/licenses/LICENSE-2.0
...
-- [Spring 2.6.x](https://github.com/flapdoodle-oss/de.flapdoodle.embed.mongo.spring/tree/spring-2.6.x)
 - [Spring 2.7.x](https://github.com/flapdoodle-oss/de.flapdoodle.embed.mongo.spring/tree/spring-2.7.x)
 - [Spring 3.0.x](https://github.com/flapdoodle-oss/de.flapdoodle.embed.mongo.spring/tree/spring-3.0.x)
+- [Spring 3.0.x](https://github.com/flapdoodle-oss/de.flapdoodle.embed.mongo.spring/tree/spring-3.1.x)
```
Comment (michaelmosmann) [inline README.md:73]: The Label should be changed to Spring 3.1.x.

**Review by michaelmosmann — COMMENTED** [review]
Thank you for your contribution. Can you agree with my remarks?

File: README.md (hunk 1 above)
Comment (izeye) [reply README.md:63]: @michaelmosmann Spring Boot still seems to manage dependency for Embedded MongoDB in 2.7.x here: https://github.com/spring-projects/spring-boot/blob/2.7.x/spring-boot-project/spring-boot-dependencies/build.gradle#L340-L346

Am I missing something?

File: README.md (hunk 2 above)
Comment (izeye) [reply README.md:71]: @michaelmosmann I didn't count its commercial support, but 2.5.x also has been EOLed even for commercial support. I'll reinstate the entry for 2.6.x, but please let me know if you still want 2.5.x to be reinstated, too.

See https://spring.io/projects/spring-boot#support

**Comment by izeye** [comment]
@michaelmosmann Thanks for the feedback! I applied it in ccb9ee4.

File: README.md (hunk 2 above)
Comment (michaelmosmann) [reply README.md:71]: @izeye I do not provide commercial support for spring 2.5.x but there are still users out there using it. So i would leave it:)

File: README.md (hunk 1 above)
Comment (michaelmosmann) [reply README.md:63]: @izeye there are supporting an old version of this library .. and stopped to upgrading it to the new version (see https://github.com/flapdoodle-oss/de.flapdoodle.embed.mongo/issues/378 if you are interested in some 'fun')

**Comment by michaelmosmann** [comment]
@izeye had to make a new release (without any changes). So i got the last part of your commit already...

**Comment by izeye** [comment]
@michaelmosmann Thanks! I updated as you suggested.

**Comment by izeye** [comment]
@michaelmosmann Sorry, I missed your last comment. Then, there's nothing left except fixing a typo that I meant to include in this PR when opening it, but somehow I missed it.

**Review by michaelmosmann — APPROVED** [review]

**Comment by michaelmosmann** [comment]
@izeye .. even if there is not much left i would like to thank for your time and effort.. :)

**Pull request merged**